server/deadfish.cpp

```diff
@@ -87,3 +87,11 @@ CollisionMask::CollisionMask(const FlatBuffGenerated::CollisionMask* fb_Col) {
 	body->CreateFixture(&fixtureDef);
 	body->SetUserData(this);
 }
+
+Tilelayer::Tilelayer(const FlatBuffGenerated::Tilelayer* fb_Tl) : width(fb_Tl->width()), height(fb_Tl->height()),
+	    tilesize(fb_Tl->tilesize()->x(), fb_Tl->tilesize()->y()) {
+    if(!(fb_Tl->tiledata()->size())) return;
+    for(auto tile : *fb_Tl->tiledata()) {
+        this->tiledata.push_back(tile);
+    }
```

Comment:
There is a `std::vector` constructor which takes a pair of iterators and copies all elements in the range ([point 5](https://en.cppreference.com/w/cpp/container/vector/vector)). Instead of writing this loop, you can do the following:

```c++
..., tiledata(fb_T1->begin(), fb_T1->end()), ...
```

This will do only one allocation (unlike the current loop) and will have the same effect (all elements from the original vector will be copied to the new one).